primitives/Cargo.toml

```diff
@@ -8,10 +8,12 @@ version = "0.0.0"
 [dependencies]
 codec.workspace = true
 scale-info.workspace = true
+sp-runtime.workspace = true
 
 [dev-dependencies]
 enum-iterator = "2.1.0"
 
 [features]
 default = [ "std" ]
-std = [ "codec/std", "scale-info/std" ]
+runtime = [  ]
```

Comment:
would recommend putting some placeholder in here to make zepter and taplo happy.

Reply:
How?